app/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="accent">#99cc00</color>
-    <color name="divider">#8A000000</color>
+    <color name="accent">@*android:color/accent_device_default_light</color>
+    <color name="primary">@*android:color/primary_device_default_settings_light</color>
+    <color name="primaryDark">@*android:color/primary_dark_device_default_settings_light</color>
+    <color name="background">@*android:color/background_device_default_light</color>
+    <color name="actionBarPrimary">@*android:color/primary_device_default_light</color>
+    <color name="statusBarColor">@*android:color/primary_dark_material_light_light_status_bar</color>
+    <color name="red">@*android:color/error_color_device_default_dark</color>
```

Comment:
I've never seen this `*` in color definitions. Is this real or a mistake?

Android Studio says it can't resolve it and marks it all red. When trying to compile without the star I get `app/src/main/res/values/colors.xml:3:5-76: AAPT: error: resource android:color/accent_device_default_light is private.`

Reply:
No Its not a mistake. values are referencing from fwb https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/res/res/values/colors_device_defaults.xml;l=46?q=color_device&sq= .

Reply:
I see plenty of AOSP apps using this (with the `*`) - but I don't know if any of them are built using gradle too (likely not)

https://cs.android.com/search?q=accent_device_default_light

Reply:
I have compiled and tested with android studio.Even It recognized the values of the colors

Reply:
I won't throw any error "unresolved symbol" in compile regardless of the reds marks in code editor